### PR TITLE
Important Update for videomega.py

### PR DESCRIPTION
**Pull request by HIGHWAY99**
Merged things to keep the current slight handling for movieshd.co, while adding more and better handling for videomega.tv..  Thanks to smokdpi for some improved videomega.tv handling code.

### CODE REVIEW

**Comment by Eldorados** [comment]
Ps. no more space removing! Don't waste your time doing that, makes trying to figure out what was changed in these VERY hard to follow with so many line changes being tracked, and makes code harder to read

**Comment by bstrdsmkr** [comment]
Just in case you didn't know, you can add ?w=1 to the review url to ignore
lines with whitespace only changes
https://github.com/HIGHWAY99/script.module.urlresolver/commit/d3656244999e5663a4b7d37fe3cf2fdcfa354428?w=1
vs
https://github.com/HIGHWAY99/script.module.urlresolver/commit/d3656244999e5663a4b7d37fe3cf2fdcfa354428
